<?xml version="1.0"?>
<launch>

  <arg name="gazebo_models" value="$(find
                                   manip_e)/data/models:$(find
                                   ur5_gazebo)/models:$(optenv
                                   GAZEBO_MODEL_PATH)" />
  
  <!-- Export env variable so that gazebo finds our models -->
  <env name="GAZEBO_MODEL_PATH" value="$(arg gazebo_models)"/>
  
  <node name="collision_check" pkg="manip_e" type="collision_check2.py"
        output="screen"/>
  
</launch>
